polkadot/roadmap/implementers-guide/src/protocol-validator-disabling.md

```diff
@@ -0,0 +1,281 @@
+# Validator Disabling
+
+## Background
+
+As established in the [approval process](protocol-approval.md) dealing with bad parablocks is a three step process:
+
+1. Detection
+1. Escalation
+1. Consequences
+
+The main system responsible for dispensing **consequences** for malicious actors is the [dispute system](protocol-disputes.md) which eventually dispenses slash events. It is important to note the **high latency** of the punishment as it is only effective after 27 eras (27 days in Polkadot). Dispute concluding by itself does not immediately remove the validator from the active validator set.
+
+> **Note:** \
+> There was an additional mechanism of automatically chilling the validator which removed their intent to participate in the next election, but the removed validator could simply re-register his intent to validate.
+
+There is a need to have a more immediate way to deal with malicious validators. This is where the validator disabling comes in. It is focused on dispensing **low latency** consequences for malicious actors. It is important to note that the validator disabling is not a replacement for the dispute or slashing systems. It is a complementary system that is focused on lighter but immediate consequences usually in the form of restricted validator privileges.
+
+The primary goals are:
+- Eliminate cases where attackers can get free attempts at attacking the network
+- Eliminate or minimize the risks of honest nodes being pushed out of consensus by getting unjustly slashed
+
+The above three goals are generally at odds so a careful balance has to be struck between them. We will achieve them by sacrificing some **liveness** in favor of **soundness** when the network is under stress. Maintaining some liveness but absolute soundness is paramount.
```

Comment:
Not sure what scalability you're referring to and spam and very much be contained in `Eliminate cases where attackers can get free attempts at attacking the network`. Spamming disputes = repeated attack attempts so we need to prevent them from being free and then all is good.